crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -243,6 +252,48 @@ where
     fn key(key_index: u16) -> Vec<u8> {
         (Self::KEY_PREFIX, key_index.to_le_bytes()).encode()
     }
+
+    /// Get segment headers that are expected to be included at specified block number.
+    pub fn segment_headers_for_block(&self, block_number: BlockNumber) -> Vec<SegmentHeader> {
+        let Some(last_segment_index) = self.max_segment_index() else {
+            return Vec::new(); // not initialized
```

Comment:
Please put comments before the thing you're commenting, not after:
```suggestion
            // Not initialized
            return Vec::new();
```

Otherwise it doesn't survive long lines and other kinds of refactoring very well.